test/js/specs/happy.js

```diff
@@ -2,125 +2,454 @@ const path = require('path')
 const expect = require('chai').expect
 const locale = (lang="en") => require(`../../../src/locales/${lang}.json`)
 import {describe, it} from '../utils/mochaw'
+import {forEach} from '../utils/misc'
+const supportedLanguage = ["en", "es"]
 
 const options = {
-  pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
+  pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload', 'DocumentUploadConfirmation', 'VerificationComplete']
 }
 
 const localhostUrl = 'https://localhost:8080/'
 
-describe('Happy Paths', options, ({driver, pageObjects}) => {
-  const {documentSelection, welcome, documentUpload} = pageObjects
+describe('Happy Paths', options, ({driver, pageObjects, until}) => {
+  const {documentSelection, welcome, documentUpload, documentUploadConfirmation, verificationComplete} = pageObjects
 
   describe('welcome screen', function () {
 
-    const copy = locale("en")
-    const welcomeLocale = copy["welcome"]
+    forEach(supportedLanguage, (lang) => {
+      const copy = locale(lang)
+      const welcomeLocale = copy["welcome"]
```

Comment:
I think this copy should be part of the page object, otherwise it breaks the architecture in my mind. unless you have strong reason to put it here

Reply:
We have tried with @stefaniacardenas to move the locale inside the page object class but this will require a heave refactoring and we don't think it's worth doing it right now as we will have to change this when we will be implementing the initialization of the SDK with options.  

If you have suggestions, let's talk offline.

Reply:
I'm not really sure why it would be such an heave refactor. This is a proposal, which seems relatively simple:
`welcome.copy('en').welcome`
`documentSelector.copy('en').passport_hint`

Also, when you change the sdk initialization this will remain the same, you still have to compare copy, so this debt won't necessarily be addressed.